cluster/v1beta1/helpers.go

```diff
@@ -108,3 +110,70 @@ func GetBoundManagedClusterSetBindings(namespace string,
 
 	return boundBindings, nil
 }
+
+type PlacementDecisionGetter interface {
+	List(selector labels.Selector, namespace string) (ret []*PlacementDecision, err error)
+}
+
+// +k8s:deepcopy-gen=false
+type PlacementDecisionClustersTracker struct {
+	placement               *Placement
+	placementDecisionGetter PlacementDecisionGetter
+	decisionClusters        sets.String
+	lock                    sync.RWMutex
+}
+
+func NewPlacementDecisionClustersTracker(placement *Placement, pdl PlacementDecisionGetter, existDecisionClusters sets.String) *PlacementDecisionClustersTracker {
+	pdct := &PlacementDecisionClustersTracker{
+		placement:               placement,
+		placementDecisionGetter: pdl,
+	}
+	if existDecisionClusters != nil {
+		pdct.decisionClusters = existDecisionClusters
+	}
```

Comment:
should you ini the decisitionClusters when existDecisionClusters is nil?

also maybe existingScheduledClusters

Reply:
fixed.